fixtures/ios_app_with_signing/Tuist/Signing/debug.p12.encrypted

```diff
@@ -0,0 +1 @@
+zef0qRuIoRTK19eeTXDhSQo/em+yOvJYMh8LbGWr247nlT6ZLwTtkdzuusm3X3/1wbZTn61Aqzd0H4Y17hFvf6eHRoOMp4kcb2ED1KbbaVgB/l7FiYh5WOxbXmC075XWXVe1a8BJsIEd7XR+xWsQpll3ruT48w9DAnHkVENhVqsGpylEoW5jgAfOfqE7Y0IW4YB3g/rCZMuTlD9Vikn+M98ePqiXMdpjGJB97LRS0KjRXijHD3AhrtnL2QKiyKdKAK4nJZHZUJEv55gsjZruamgvviJ0LM5p+SNAMZAsKqIX8T5dMUcyr1X2wIFh6b+/lyjv3JIuRVpoXLxQJMEJ+0Bhh0oCDfxZ5c1uwujPl01BoLfHf42On2QQ4GV/mk0XADGF9URv4TatE9BWAdP5PERgG0roFAkivCizPyNBtI5xq1xiWpXnFdc/Qhy6QpqYr2VPuVWK/WaTI9ZLOT9UpmgQ36JhrZc3JUjVxNhTiUiD3bgpsFcaoRSoUoPBx7GSSG44b6j+sVdMTRQ6BusE4e6MYFZRxn2jmvn1YMUPJ+BC6UmjvjGaO7gOIULnjeUDfrU1CRvvZWPkjoS1a6l9vsyYno4sJ9JRhuxjTCPdDUp1k2IKk3cFE3j9NjsRWjAvfu43CZ1F83RgA+XRI7udfJoaMzAvq4I1OyMw8/sXAlH0tZZhUXuIXhq3SUHDYXT5QB4+eVE03vWHJUattYS57IskElWUsPdf7dIdUQFQxHeKnSDvlRfWQMqL0btwRUvHV54HY/VOl0RVrSMTekJSzAtAoX/Jqur60QvbEZrV5i/QM0i7TkeqrU3IEP5rRNmBGy3LIRthDerChZ3Tj0ZnJO7xlEYpagIsm3JXXvn/5vF8RkU/h99d9Oo8x/cRT6cKGtdtVblUr9+cRw9AHvvDt3c2wmLwdlFORUZD/ORPMEHipgB52YYmjua16x7B42yR+GoqL/WYCxnnuDjekdrl9P20dkivMaoezrIqCUSzMYEbFR84ob1Z9fRn9stKGaGEVdROmnu/C5SA7VpY3BprJYx+Q2l7zFneA+MRX7bsk1pj7jHTeLVsp9NnRjYg38umsLCoNy4kdprDWckbV+H2PBkYCr96JVEFX8DvG8ElJ82Qk45h7QiVnjzczlufD4vxtlD3qv3mMjedr/8Xax429WTEBhDM/R5hG70sjJGP1sPV67OzII4HqFRnAufn71MUisNeqc7hipJqvTIrPgYH8ThUORs6dVynmKSURUdI4FBzkIStSttf1XIO6XH+Iew4y/ERXyULOOtt4+cBWPXy3nHtybqhb6gXkyzHLXiy82uimUVhvS08WOc1oQXTfPCM+8uHyKFnevvd5Z61S31QN4l/hX3ZxIIiHAaO9SM9rvVLSZptf1Zuo3GhDomtpa22qtFjUSdD5BmQuX3g9JI45tv4LAtA4yRGHm974015k+sGP5HZTH990CLPpK4Pfr4oYQqx0gtb54gd+8A+wrG82clYC6C5r0NtCV6Ov3iVPCA+ONGEAfuw3u7E4gQMYXXwKrfbvcDHi1Z5hgjGmXofacnJOZse5qluFpyxl8n3sjul+br0WMlrVX4j4apxPUpHakTScv6RW4Q0hKV2p6dlkeyv86KieEHCmbcY1fzXyvL8M9mjsqrnSosVppO9SvGGdnSYVIm9d/lgTZRMdvbBp19SCXN9XaIL6VF1tvpgo3iYD+THNUbppVv2cgcKFPY1BWFUFVWxBA3tzE807w0t/eV3Db4oNAtpp+Ej81hcnEaWju9hNTziLjWRSVEYX0zsbB48Jhq9gvjmeuKSGN6FhGQgoFl3Sv+P9Qnsf+Rkt94+/RVe7yrJP6+DUMPubC/u3n5P8JzcPPi4wYEt91sRAJAzQGtigzt/jQIsLXNEeZetlpibAuEudDEUGE7yh0t5sSCZdlgqEsEEvZ3QkgI+W2XfTowX5ci2PXyMpFo2U7rAyHVkioTtpcVKKAK4uPRFOP9Vaz6SeLh7Xh/Hs5jCf6/fmLbPh+sKUR2JV4VYfCC98BTqRvNcqa4r5L5kRjr7DI5VkOJNwUNK20ny/DNjEx9d6h3lyTvYFNuDQ4Bz9ZtIjXU/kCAccYjfnUmTDZKLU5Uxyi6Dsx960CRvbCDnxWGBhT/xzGPqZ4lXhs/7vo2h7DsSDvK3TzQUDOE4O2AailHcOnaXfY81GtzjI0i5ySVtq+2EyhkmialgXJM3EDq8l74s3TcN5WaLmYmwNKZmycJQC5r2EDqGwRIzb5dDpZJUcU2sN6ouYRtqA3Aifs5Pj0Atd7249rFQ+7FVA/uOzb4Wy6pXmxYDpeBl4zxymdpyYtULRNbIlbQxQcZRu6TYBvs8aOAcIpsa8ZyA1wK/aDRTnSrFSJjyb7UzsHWlIuE65mX5aPkenSwN495ymvPD8ajy5EJUaBcHxs7MbE1Pn0RMv7jYBsxY2qPre4QNv44lnFqgXND0vziwUtf7v1pY2hFCDsA1XQpQItk3m9yKwMbR/CF7FyExyeWY6AQ6da8PxD+Hr+zCl/T2IRRMDgrGnaX8wp46tdwEyaG6cYZt1LwHwS/VsjMb1Ng41BqlgUXDdRdJmJ2tX5G/uCAeP1nrtOtJP3jqCRAxUUOvYnARHNrS64ePg2BWzrtQJEHgMZydq9NjXEax13BePP5C47NBWkiE32QmlRKP6D40yoDQHmqBLUyIt4AbdZgX+EJOWOBVHZaIh6bRO44Bm9gSxHwEnuGPSVcF/neiamCrSFahCrr+V6H6Bt6+68vaJUn5d+c52sYK5xylzFVcMS8vZU/Czn/gjdff0mvo3j4iRmTaa2E/n+C+TpPhtGUwP8z30+TKiGGwKsOKLeBDPZstAaeeTxbNk4UfX8vUvIWkijnzkERsJXBODnI9wIzIAVVJl/aGTtrSimO2eHJb1mnUSKd1GG1HHepuQetrJOAj/EZcncsyne3j/OfRB6Mzf2/tvXn+8pIruQ0MDKgzaqqY5HDA+pOT1svYg==
```

Comment:
@fortmarek how do you know the target this profile is for? `debug.p12`. Is that something that you read from the profile?

Reply:
I found it! So we get the name from the content of the profile. What about requiring the name of the file to follow a convention? For example, looking at the name of the file int the fixture, `debug.p12` I can't tell what target the profile is for.

Reply:
I have modified the convention, feel free to have a look to the edited fixture